server/api.js

```diff
@@ -1,12 +1,54 @@
 import { Router } from "express";
-
+// import { Pool } from "pg";
+import db from "./db";
 import logger from "./utils/logger";
 
+
 const router = Router();
 
-router.get("/", (_, res) => {
-	logger.debug("Welcoming everyone...");
-	res.json({ message: "Hello, world!" });
+// router.get("/", (_, res) => {
+// 	logger.debug("Welcoming everyone...");
+// 	res.json({ message: "Hello, world!" });
+// });
+
+router.get("/energizers", async (req, res) => {
+	try {
+		const result = await db.query("SELECT * FROM energizers");
+		res.json(result.rows);
+	} catch (error) {
+		logger.error(error);
+		res.status(500).json(error);
+	}
+});
+
+router.post("/energizers", async (req, res) => {
+	const { name, description } = req.body;
+
+	if (!name || !description) {
+		return res.status(400).json({ error: "Missing required fields" });
+	}
+
+	try {
+		await db.query(
+			"INSERT INTO energizers( name, description) VALUES ( $1, $2)",
+			[ name, description]
+		);
+		const result = await db.query("SELECT * FROM energizers");
+		res.status(200).send(result.rows);
```

Comment:
I don't think this would be normal expectation to return all of the energisers from a POST request - the client would probably make another GET if that is necessary. It would be a good idea to return at least the id of the new energizer or the whole object (but in same format as the new api method that returns individual energizers now on main branch).